sig-scalability/slos/pod_startup_latency.md

```diff
@@ -0,0 +1,47 @@
+## Pod startup latency SLI/SLO details
+
+### User stories
+- As a user of vanilla Kubernetes, I want some guarantee how quickly my pods
+will be started.
+
+### Other notes
+- Only schedulable and stateless pods contribute to the SLI:
+  - If there is no space in the cluster to place the pod, there is not much
+    we can do about it (it is task for Cluster Autoscaler which should have
+    separate SLIs/SLOs).
+  - If placing a pod requires preempting other pods, that may heavily depend
+    on the application (e.g. on their graceful termination period). We don't
+    want that to contribute to this SLI.
+  - Mounting disks required by non-stateless pods may potentially also require
+    non-negligible time, not fully dependent on Kubernetes.
+- We are explicitly excluding image pulling from time the SLI. This is
+because it highly depends on locality of the image, image registry performance
```

Comment:
Sorry to jump in this PR! Should you specify that this timing could vary based on container runtime ? Also, when you mean Kubernetes default installation means you use Docker as container runtime. Thanks

Reply:
> Should you specify that this timing could vary based on container runtime ?

That may matter for SLO (the fact that some runtimes are slower than other doesn't affect SLI - i.e. what we measure).

For SLO - we assume default k8s installation, which is Docker.

So yes, while the container runtime matters, it's not important here.

Reply:
@wojtek-t Thanks for the clarification